src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.type.Color;
+
+public class ColorSupplier extends BasicRandomValueSupplier {
```

Comment:
The ColorSupplier class should not extend BasicRandomValueSupplier. Instead, you can create an instance of the BasicRandomValueSupplier class inside the ColorSupplier class to use its functionality.